datastore.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: datastore-dev
```

Comment:
`dev` or `staging`?

Should ensure names are consistent.

Reply:
dev

Reply:
done